osf_tests/test_file_metadata.py

```diff
@@ -31,6 +32,10 @@ def inject_placeholder_doi(json_data):
 @pytest.mark.django_db
 class TestFileMetadataRecordSerializer:
 
+    @pytest.fixture(autouse=True)
+    def datacite_file_schema(self):
+        return ensure_datacite_file_schema()
+
```

Comment:
Ditto, moving out of the class and run only once in [osf_tests/test_file_metadata.py](https://github.com/CenterForOpenScience/osf.io/pull/9994/files#diff-1d6537b166c843d95abf4bdb8f0cbef247e628b0d428afe14a8b9c2d744dbfc8)